galoy-client/src/client/queries.rs

```diff
@@ -0,0 +1,209 @@
+use graphql_client::GraphQLQuery;
+
+use crate::GaloyClientError;
+
+use self::stablesats_wallets::{StablesatsWalletsMeDefaultAccountWallets, WalletCurrency};
+
+pub type SafeInt = i64;
+
+#[derive(GraphQLQuery)]
+#[graphql(
+    schema_path = "src/client/graphql/schema.graphql",
+    query_path = "src/client/graphql/mutations/user_request_auth_code.graphql",
+    response_derives = "Debug, PartialEq, Eq"
+)]
+pub struct StablesatsAuthCode;
+pub type Phone = String;
+pub type StablesatsAuthenticationCode = stablesats_auth_code::StablesatsAuthCodeUserRequestAuthCode;
+impl TryFrom<stablesats_auth_code::ResponseData> for StablesatsAuthenticationCode {
+    type Error = GaloyClientError;
+
+    fn try_from(response: stablesats_auth_code::ResponseData) -> Result<Self, Self::Error> {
+        let auth_code = response.user_request_auth_code;
+        if let Some(is_success) = auth_code.success {
+            if !is_success {
+                return Err(GaloyClientError::Authentication(
+                    "Authentication failed".to_string(),
+                ));
+            } else {
+                return Ok(auth_code);
+            }
+        }
+
+        Err(GaloyClientError::Authentication(
+            "Empty authentication code".to_string(),
+        ))
+    }
+}
+
+#[derive(GraphQLQuery)]
+#[graphql(
+    schema_path = "src/client/graphql/schema.graphql",
+    query_path = "src/client/graphql/mutations/user_login.graphql",
+    response_derives = "Debug, PartialEq, Eq, Clone"
+)]
+pub struct StablesatsUserLogin;
+pub type AuthToken = String;
+pub type OneTimeAuthCode = String;
+pub type StablesatsLogin = stablesats_user_login::StablesatsUserLoginUserLogin;
+impl TryFrom<stablesats_user_login::ResponseData> for StablesatsLogin {
+    type Error = GaloyClientError;
+
+    fn try_from(response: stablesats_user_login::ResponseData) -> Result<Self, Self::Error> {
+        let user_login = response.user_login;
+        Ok(user_login)
+    }
+}
+
+#[derive(GraphQLQuery)]
+#[graphql(
+    schema_path = "src/client/graphql/schema.graphql",
+    query_path = "src/client/graphql/queries/transactions_list.graphql",
+    response_derives = "Debug, PartialEq, Clone"
+)]
+pub struct StablesatsTransactionsList;
+pub type WalletId = String;
+pub type Timestamp = u64;
+pub type Memo = String;
+pub(crate) type SignedAmount = f64;
```

Comment:
please use `Decimal` for all money amounts. Float should NEVER be used for money! It is imprecise.